src/Illuminate/Console/Scheduling/ManagesFrequencies.php

```diff
@@ -274,6 +274,31 @@ public function monthlyOn($day = 1, $time = '0:0')
     }
 
     /**
+     * Schedule the event to run on the last day of the month at a given time.
+     *
+     * @param  string  $time
+     * @return $this
+     */
+    public function endOfMonth($time = '0:0')
+    {
+        return $this->dailyAt($time)->when($this->isEndOfMonth());
+    }
+
+    /**
+     * Schedule the event to run on the last day of the month.
+     *
+     * @return \Closure
+     */
+    private function isEndOfMonth()
+    {
+        return function () {
+            $now = Carbon::now();
+
+            return $now->day() == $now->daysInMonth();
```

Comment:
Corrected.
 
I do that *every* *single* *time* with Carbon. Gah. Hats the method/property name overlap, we do.